ZenPacks/zenoss/Microsoft/Windows/WinService.py

```diff
@@ -164,10 +164,33 @@ def get_winservices_modes(self):
             if svc.monitored():
                 data[svc.id] = {'modes': svc.get_serviceclass_startmodes(),
                                 'mode': svc.startMode,
-                                'monitor': svc.monitored()
+                                'monitor': svc.monitored(),
+                                'severity': svc.getFailSeverity(),
                                 }
         return data
 
+    def getTemplateDefaultService(self):
+        '''Return datasource for template if it exists'''
+        template = self.getRRDTemplate()
+        if template:
+            return template.datasources._getOb('DefaultService', None)
```

Comment:
Also, the service will always have a template associated with it.  You should instead save the datasource name as an attribute and return the severity on it

something like this:
before line 98:
`self.datasource_name = ''`

before returning True on lines 104, 109, then set the name
`self.datasource_name = datasource.id`

then your new method would be 
```
getTemplateDataSource(self):
    if self.datasource_name:
        template = self.getRRDTemplate()
        return template.datasources._getOb(self.datasource_name, None)
```